### PR TITLE
Fixes #10452: GUI breaks completely with IE 11

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/10452

### CODE REVIEW

File: rudder-web/src/main/resources/rudder-apache-common.conf [hunk 1]
```diff
@@ -4,6 +4,9 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
+# Force using latest mode in IE.
+Header add X-UA-Compatible "IE=Edge"
```
Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-apache-common.conf]: Shouldn't we add it only in interface specific urls, like 2 lines above ?

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Indeed, I was testing the cpondition.

File: rudder-web/src/main/resources/rudder-apache-common.conf [hunk 2]
```diff
@@ -4,6 +4,11 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
+# Force using latest mode in IE.
+<LocationMatch "^/(?!rudder/api)">
```
Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-apache-common.conf]: I still think this should match the location above.
By changing one or the other

Comment (amousset) [inline rudder-web/src/main/resources/rudder-apache-common.conf]: The other one is specific to Rudder "autoreload" on error feature. We could exclude the API (it is done in 4.1), but there is not reason to apply it on the manual or ncf-builder.

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**